docs/source/release/v4.1.0/idfs.rst

```diff
@@ -0,0 +1,13 @@
+==================================
```

Comment:
Can we put these in `framework.rst`? We generally put the IDF changes in their own section in there tends not to be too many for its own page.